### PR TITLE
Image and rendering fixes

### PR DESCRIPTION
**Pull request by mmattel**
This PR fixes the following:

* Nearly all images needed a sizing fix to improve rendering
* Image positioning
* Table centering where necessary
* Table fixes
* Removing unneeded index.adoc files from the navigation

No content changes.

Backport to 5.2

### CODE REVIEW

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -171,7 +170,7 @@ Keeping software up to date is crucial for file integrity and security – if so
 
 The ownCloud Desktop App talks to a server, e.g. the ownCloud server, so you do not only have to upgrade your Desktop App when there is a new version for it, also the server has to be kept up-to-date by your sysadmin. Starting with version 2.5.0, the Desktop App will show a warning message if you connect to an outdated or unsupported server:
 
-image::faq/oc-unsupported-version-warning-message.png[image, width=600,pdfwidth=60%]
+image::faq/oc-unsupported-version-warning-message.png[image, width=350,pdfwidth=60%]
```
Comment (phil-davis) [inline modules/ROOT/pages/faq.adoc:173]: how do you know when to put 1 colon after "image", and when to put 2 colons?

**Review by phil-davis — APPROVED** [review]
Approving so you can merge if you are OK with this. But I don't understand why "image::" was changed to "image:", but not in all cases.

**Comment by mmattel** [comment]
In a nutshell:

image: adds the image "inline" with text, means it aligns where the next character would be.

image:: always centers the text in the frame (page or table etc), used where the image stands on its own.

**Comment by mmattel** [comment]
@phil-davis fyi: https://docs.asciidoctor.org/asciidoc/latest/macros/images/

**Pull request merged**